USAGE.md

```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
 instead expands the selection by that much movement similar to visual
 mode in vi.
 
--   `h`: move cursor left one char
--   `j`: move cursor down one char
--   `k`: move cursor up one char
--   `l`: move cursor right one char
--   `w`: move cursor to next word
--   `b`: move cursor to previous word start
--   `$`: move cursor to end of line
--   `^`: move cursor to start of line
--   `G`: go to line, or end of file with no prefix
+-   `h`: move left by columns
+-   `j`: move down by lines
+-   `k`: move up by lines
+-   `l`: move right by columns
+
+-   `w`: move to next word
+-   `b`: move back word
+-   `$`: move to last column
+-   `^`: move to first character
+-   `G`: move to line, or last line without a prefix
 
 Add new text by entering Insert mode with `i` or `a`.
 
--   `i`: insert before current selection
--   `a`: insert after current selection
+-   `i`: insert before cursor
+-   `a`: append after cursor
 
 Delete existing text can either happen with shorthands like Backspace or
 `x`, or with full Delete mode.
 
 -   `d`: enter Delete mode
--   `dd`: delete current line
--   `x`: delete current char
--   Backspace: delete durrently selected text (or previous character if
-    nothing is selected)
+-   `x`: delete character
+-   Backspace: delete currently selected text, or previous character if
+    nothing is selected
 
 Some other common commands:
 
--   `p`: Paste
--   `u`: Undo
--   `;`: Give keyboard focus to tagline (see below for tagline usage)
--   `:`: Give keyboard focus to end of tagline (see below)
--   Up: scroll viewport up half a screen
--   Down: scroll viewport down half a screen
--   Esc: Save if buffer has been modified, Exit if buffer has not been
+-   `p`: paste
```

Comment:
the `u` for Undo disappeared, and Paste is a builtin-command (but paste isn't), so it's better to keep it capitalized so that the word is executable in the documentation.